_episodes/04-advanced-layer-types.md

```diff
@@ -502,6 +502,31 @@ It seems that the model is overfitting somewhat, because the validation accuracy
 > audio data for speech recognition, or 3d structures of chemical compounds.
 {: .callout}
 
+> ## Why and when to use convolutional neural networks
+>
+> 1. Would it make sense to train a convolutional neural network (CNN) on the penguins dataset and why?
+> 2. Would it make sense to train a CNN on the weather dataset and why?
+> 3. (Optional) Can you think of a different machine learning task that would benefit from a
+>    CNN architecture?
+>
+> >
+> > ## Solution
+> > 1. No that would not make sense. Convolutions only work when the data is ordered
```

Comment:
Not just spatial. It can be temporal as well. Maybe we can rephrase it to "Convolutions only work when the features of the data can be ordered in a meaningful way. Pixels are ordered in a spatial dimension. This kind of order cannot be applied to the features of the penguin dataset.